tools/pd-ctl/pdctl/command/config_command.go

```diff
@@ -450,7 +451,41 @@ func NewPlacementRulesCommand() *cobra.Command {
 		Run:   deleteRuleGroupFunc,
 	}
 	ruleGroup.AddCommand(ruleGroupShow, ruleGroupSet, ruleGroupDelete)
-	c.AddCommand(enable, disable, show, load, save, ruleGroup)
+	ruleBundle := &cobra.Command{
+		Use:   "rule-bundle",
+		Short: "process rules in group(s), set/save perform in a replace fashion",
+	}
+	ruleBundleGet := &cobra.Command{
+		Use:   "get <id>",
+		Short: "get rule group config and its rules by group id",
+		Run:   getRuleBundle,
+	}
+	ruleBundleGet.Flags().String("out", "", "the output file")
+	ruleBundleSet := &cobra.Command{
+		Use:   "set",
+		Short: "set rule group config and its rules from file",
+		Run:   setRuleBundle,
+	}
+	ruleBundleSet.Flags().String("in", "bundle.json", "the file contains one group config and its rules")
```

Comment:
could `group.json` be better?

Reply:
Not bad. See the chat, I left a message.